jax/_src/config.py

```diff
@@ -54,6 +54,14 @@ def int_env(varname: str, default: int) -> int:
   return int(os.getenv(varname, str(default)))
 
 
+UPGRADE_BOOL_HELP = (
+    " This will be enabled by default in future versions of JAX, at which "
+    "point its reverse setting will be considered deprecated. In a version "
```

Comment:
Is "reverse setting" clear?

Reply:
I wonder if we can just link to a deprecation policy docs page here.

Reply:
Good idea. Done.